### ISSUE TITLE
predict=True/False and randomize_length for different prediction length

### ISSUE THREAD

**Issue by GNuzzarello** [jdb78/pytorch-forecasting#1179]
- PyTorch-Forecasting version: 0.10.2
- PyTorch version:
- Python version: 3.8.5
- Operating System: Windows

I want to train a TFT model to do forecasting of different length. I set min_prediction_length=1 and max_prediction_length=16 but my question is:
1) randomize_length create samples of different length?
2) Should I set predict = False in TimeSeriesDataSet.from_dataset to create a validation set with different prediction length and not only max_prediction_length?? 

```
# create validation set (predict=True) which means to predict the last max_prediction_length points in time
# for each series
validation = TimeSeriesDataSet.from_dataset(training, myData, predict=False, stop_randomization=True)
```
I hope I was clear. Thank you

**Comment by sairamtvv** [comment]
validation = TimeSeriesDataSet.from_dataset(training, data, predict=predict,
                                                    stop_randomization=True,
                                                    min_prediction_idx=training_cutoff + 1)

What is generally do is,  during training I make predict = False  but during prediction I make predict=True. But dont forget to add min_prediction_idx. This is what I do and hope this helps
